tests.sh

```diff
@@ -14,8 +14,8 @@ then
 else
     c-bindings-tests/run.sh
 
-    $PYTHON_COMMAND setup.py build
-    $PYTHON_COMMAND setup.py test
+    $PYTHON_COMMAND setup.py build --build-platlib .
```

Comment:
Can you add a comment explaining why the `--build-platlib` option is needed?

Reply:
If we run `setup.py build`, it will build the package underneath 'build/'. `--build-platform` allows us to build the required `shared library object` to the current folder in order to run `pytest` without this error: `ImportError: No module named '_gojsonnet'`